docs/consume-packages/Package-Restore.md

```diff
@@ -115,6 +115,15 @@ To enable or disable Package Restore for all users on a computer, a developer or
 > [!Important]
 > If you edit the `packageRestore` settings directly in `nuget.config`, restart Visual Studio, so that the **Options** dialog box shows the current values.
 
+### Choose default package management format
```

Comment:
Feel like this belongs somewhere in the installing a package with visual studio page as well.